signer/src/proto/convert.rs

```diff
@@ -1593,14 +1593,25 @@ impl From<DkgPublicShares> for proto::SignerDkgPublicShares {
             dkg_id: value.dkg_id,
             signer_id: value.signer_id,
             commitments: value.comms.into_iter().map(|v| v.into()).collect(),
+            kex_public_key: Some(proto::Point::from(value.kex_public_key)),
         }
     }
 }
 
 impl TryFrom<proto::SignerDkgPublicShares> for DkgPublicShares {
     type Error = Error;
     fn try_from(value: proto::SignerDkgPublicShares) -> Result<Self, Self::Error> {
+        let kex_public_key = match value.kex_public_key {
+            Some(key) => Point::try_from(key)?,
```

Comment:
I just realized that there is a simpler way. We should:
1. Create a new protobuf that is like the existing `proto::SignerDkgPublicShares` type.
2. Create a new local-to-signer crate type that is like the existing `wsts::net::DkgPublicShares` type.
3. Use the new types for serializing and deserializing to and from the database.
4. Add implementations for transforming the new protobuf type to the new local `DkgPublicShares` type.
5. Require the `kex_public_key` here.

This seems like a cleaner thing than we are currently going for, even though it's more complex. I don't mind doing the leg work on this one, but before I do so, do you have any thoughts on this?